packages/node/src/kernel.js

```diff
@@ -15,6 +15,9 @@ const nativeRequire = require
 module.exports = { createKernel }
 
 function createKernel ({ projectRoot, lavamoatPolicy, canonicalNameMap, debugMode, statsMode, scuttleGlobalThis, scuttleGlobalThisExceptions }) {
+  if (scuttleGlobalThis?.scuttler) {
+    throw new Error('Lavamoat - "scuttleGlobalThis.scuttler" option is not supported for node environment.')
```

Comment:
why not?

Reply:
* we decided to allow using "scuttlerName" config option in node env
* since it's experimental, we decided to allow using it similar to how it's used for browser envs (by defining a prop on globalRef)
* later on we'd like to allow passing an actual reference instead of a string prop
* fix 6bd2c5230